portal-impl/src/com/liferay/portal/servlet/filters/virtualhost/VirtualHostFilter.java

```diff
@@ -312,18 +312,16 @@ protected void processFilter(
 					return;
 				}
 
+				String homeURL = PortalUtil.getRelativeHomeURL(
+					httpServletRequest);
```

Comment:
@tototrinh 
This variable is only used in one specific scope, we don't need to move it out here.